src/types.ts

```diff
@@ -620,6 +622,50 @@ export enum AccountTransactionType {
     TransferWithScheduleWithMemo = 24,
 }
 
+export interface DeployModulePayload {
+    /** Version of the wasm module */
+    version: number;
+
+    /** Wasm module to be deployed */
+    content: Buffer;
+}
+
+export interface InitContractPayload {
+    /** µGTU amount to transfer */
+    amount: GtuAmount;
+
+    /** Hash of the module on chain */
+    moduleRef: ModuleReference;
+
+    /** Name of init function including */
+    initName: string;
```

Comment:
Please rename this to `contractName` and then make it so that the serialization of the payload automatically adds the `init_` before serialization. This will make it nicer for the user of the SDK, as they don't have to know about this naming convention.